oval_graph/client.py

```diff
@@ -201,18 +201,21 @@ def prepare_parser(self):
         self.parser = argparse.ArgumentParser(
             description=self.MESSAGES.get('description'))
         self.parser.add_argument(
+            '-a',
             '--all',
             action="store_true",
             default=False,
             help="Process all matched rules.")
         self.parser.add_argument(
+            '-p',
             '--hide-passing-tests',
             action="store_true",
             default=False,
             help=(
                 "Do not display passing tests for better orientation in"
                 " graphs that contain a large amount of nodes.(Not implemented)"))
```

Comment:
If it is not implemented, don't have it here, just comment it out.

Reply:
This option has been implemented, but I forgot to remove it.